roles/opcap_tool/tasks/build.yml

```diff
@@ -1,34 +1,29 @@
 ---
-- name: "Clone opcap repository"  # noqa: latest[git]
+- name: Clone opcap repository  # noqa: latest[git]
```

Comment:
If we download directly the binary from GH, it is not necessary to git clone the repository anymore right?

Reply:
no, nice catch! Let me remove it, too